independent-publisher-connectors/GitLab/readme.md

```diff
@@ -0,0 +1,41 @@
+# GitLab
+GitLab is an alternative to GitHub and Azure DevOps.  GitLab offers a robust suite of APIs for automating CI/CD pipelines.  This connector allows a user to orchestrate basic DevOps workflows within Power Apps, Logic Apps, and Power Automate.
+
+## Publisher: Roy Paar
+
+## Prerequisites
+You must provision an API Key within your GitLab account.
+
+## Supported Operations
+### Get Version
+Retrieve version information for this GitLab instance. Responds 200 OK for authenticated users.
+
+### Create a Project
+Create a new project owned by the authenticated user.
+
+### Fork a Project
+Fork a project into the user namespace of the authenticated user or the one provided.
+
+### Compare a branch, tag, or commit
+Get the diff of a branch, tag, or commit in a project.
+
+### Create a Branch
+Create a new branch in the repository.
+
+### Create a Commit
+Create a commit by posting a JSON payload.
+
```

Comment:
No Obtaining Credentials section. Please refer to this [link](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md).

Reply:
Fixed